### DISCUSSION TITLE
how to change summary click color

### DISCUSSION THREAD

**Discussion by end3r-man** [saadeghi/daisyui#3284] [Q&A] [answered]
hello, in trying to change the click color of the menu drop down color 

```html
<summary class="pl-2 pr-4 hover:bg-base-100 hover:text-primary">
```
im trying to change the color to base-100 its work for hover is i add focus:bg-base-100 its not working is their any way to change the click color??

**Comment by stabldev** [comment, marked as answer]
hey @end3r-man ,
https://tailwindcss.com/docs/hover-focus-and-other-states#hover-focus-and-active
is this what you meant? `active:`

**Reply by end3r-man** [reply]
> hey @end3r-man , https://tailwindcss.com/docs/hover-focus-and-other-states#hover-focus-and-active is this what you meant? `active:`

yep this it, tnx
